src/components/FullDetailView/FullDetailPage.js

```diff
@@ -0,0 +1,170 @@
+// Copyright (c) Microsoft Corporation and others. Licensed under the MIT license.
+// SPDX-License-Identifier: MIT
+
+import React, { Component } from 'react'
+import { bindActionCreators } from 'redux'
+import { connect } from 'react-redux'
+import { Grid, Button } from 'react-bootstrap'
+import PropTypes from 'prop-types'
+import Modal from 'antd/lib/modal'
+import 'antd/dist/antd.css'
+
+import { uiInspectGetDefinition, uiInspectGetCuration, uiInspectGetHarvested, uiNavigation } from '../../actions/ui'
+import { curateAction } from '../../actions/curationActions'
+import { ROUTE_DEFINITIONS } from '../../utils/routingConstants'
+import EntitySpec from '../../utils/entitySpec'
+import Contribution from '../../utils/contribution'
+import ContributePrompt from '../ContributePrompt'
+import FullDetailComponent from './FullDetailComponent'
+
+/**
+ * Component that renders the Full Detail View as a Page or as a Modal
+ * based on modalView property
+ *
+ */
+export class FullDetailPage extends Component {
```

Comment:
This is not necessarily a "page" rather, it is a view where it may be embedded in a modal or just be a standard component. Seems like there is relatively little shared code here. Much of it relates to either the modal case (onClose, ...) and the other parts are only used in the standard case. 

Consider splitting into two?

Reply:
Originally this was splitted into 2 components.
But since the methods for connect to the Redux store are the same (for saving, contribution), i've preferred to merge into one component, to keep it easily maintainable

Reply:
The model does not have contribute and the standard view should not have save. I'm actually not sure what code would be duplicated between the two if they were separate.